crates/viewer/re_data_ui/src/instance_path.rs

```diff
@@ -296,31 +305,53 @@ fn preview_if_image_ui(
                 .ok()
                 .flatten()
         });
-
-    let image = ImageInfo {
-        buffer_row_id,
-        buffer: image_buffer.0,
-        format: image_format.0,
-        kind,
+    let value_range = component_map
+        .get(&components::Range1D::name())
+        .and_then(|colormap| {
+            colormap
+                .component_mono::<components::ValueRange>()
+                .transpose()
+                .ok()
+                .flatten()
```

Comment:
isn't this the same?

```suggestion
                .component_mono::<components::ValueRange>()?
                .ok()
```

Reply:
ah, yes. much better!